consensus/src/sync/live/block_queue/mod.rs

```diff
@@ -267,11 +261,14 @@ impl<N: Network> BlockQueue<N> {
         let mut parent_block_number = block_number - 1;
 
         // Insert block into buffer. If we already know the block, we're done.
-        let block_known = self.insert_block_into_buffer(block, pubsub_id.clone());
-        log::trace!("Buffering block #{}, known={}", block_number, block_known);
-        if block_known {
+        if !self.insert_block_into_buffer(block, pubsub_id.clone()) {
+            log::trace!(
+                "Not buffering block #{} - already known or exceeded the per peer limit",
+                block_number
```

Comment:
Use `tracing` style:
```suggestion
                block_number
                "Not buffering block - already known or exceeded the per peer limit",
```